### PR TITLE
BAH-2665 | Create Helm chart to deploy Metabase with Bahmni docker to Dev Lite

### PR DESCRIPTION
**Pull request by sindhura-kaparthi**
## Requirements

- [X] This PR has a proper title that briefly describes the work done
- [X] I have squashed / amended the comments to make it more readable
- [X] I have included link to all the JIRA ticket('s)
- [X] I wrote the code as a pair or atleast performed a self-review of my own code
- [ ] I have updated the documentation on [Bahmni Wiki](https://bahmni.atlassian.net/wiki/spaces/BAH/overview) or [README](https://github.com/Bahmni/bahmni-metabase/blob/main/README.md) (if applicable)

## Summary
Create Helm chart to deploy metabase with Bahmni docker to Dev Lite

## JIRA tickets
https://bahmni.atlassian.net/browse/BAH-2665

### CODE REVIEW

File: .github/workflows/bahmni_metabase_build_publish.yaml [hunk 1]
```diff
@@ -35,4 +37,38 @@ jobs:
           context: package/docker/
           no-cache: true
           push: true
-          tags: bahmni/bahmni-metabase:${{env.ARTIFACT_VERSION}},bahmni/bahmni-metabase:latest
\ No newline at end of file
+          tags: bahmni/bahmni-metabase:${{env.ARTIFACT_VERSION}},bahmni/bahmni-metabase:latest
+
+      - name: Update Version and Image Tag
+        run: |
+          yq --inplace '.image.tag = "${{ env.ARTIFACT_VERSION }}"' $HELM_CHART_PATH/values.yaml
```
Comment (pajmera4) [inline .github/workflows/bahmni_metabase_build_publish.yaml]: values.yaml file is not present in the path (package/helm)

Comment (mradul1807) [reply .github/workflows/bahmni_metabase_build_publish.yaml]: @pajmera4 I guess you are checking in MAIN branch. These changes are in BAH-2665.

Comment (pajmera4) [reply .github/workflows/bahmni_metabase_build_publish.yaml]: No I was looking into the same PR. There was a confusion, values.yaml is present in the same dir

File: .github/workflows/bahmni_metabase_build_publish.yaml [hunk 2]
```diff
@@ -4,7 +4,7 @@ on:
     branches: main
```
Comment (pajmera4) [inline .github/workflows/bahmni_metabase_build_publish.yaml]: we can keep the name as build_publish.yml itself.

File: .github/workflows/bahmni_metabase_build_publish.yaml (hunk 1 above)
Comment (mradul1807) [reply .github/workflows/bahmni_metabase_build_publish.yaml]: Okay. My bad there.

File: .github/workflows/bahmni_metabase_build_publish.yaml [hunk 3]
```diff
@@ -13,9 +13,11 @@ jobs:
       - name: Trivy Scan
         run: ./.github/trivy_scan.sh
 
-  docker-build-publish:
+  build-publish-docker-helm:
     name: Docker Build & Publish
```
Comment (pajmera4) [inline .github/workflows/bahmni_metabase_build_publish.yaml]: Please include helm also here in the name

File: .github/workflows/bahmni_metabase_build_publish.yaml [hunk 4]
```diff
@@ -35,4 +37,38 @@ jobs:
...
+
+      - name: Checkout Charts Repository
+        uses: actions/checkout@v2
+        with:
+          repository: Bahmni/helm-charts
+          ref: gh-pages
+          path: helm-charts
+          persist-credentials: false
+
+      - name: Copy Helm Archive
```
Comment (pajmera4) [inline .github/workflows/bahmni_metabase_build_publish.yaml]: Rename to Copy Helm Chart.

File: package/helm/templates/configMap.yaml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Chart.Name }}-config
+data:
+  MB_HOST: "{{ .Values.config.MB_HOST }}"
+  MB_PORT: "{{ .Values.config.MB_PORT }}"
```
Comment (pajmera4) [inline package/helm/templates/configMap.yaml]: As discussed, We don't need to configure MB_HOST and MB_PORT from outside now.

File: .github/workflows/bahmni_metabase_build_publish.yaml (hunk 2 above)
Comment (sindhura-kaparthi) [reply .github/workflows/bahmni_metabase_build_publish.yaml]: Yes Done , Thank you.

File: .github/workflows/bahmni_metabase_build_publish.yaml (hunk 3 above)
Comment (sindhura-kaparthi) [reply .github/workflows/bahmni_metabase_build_publish.yaml]: Yes Done , Thank you

File: .github/workflows/bahmni_metabase_build_publish.yaml (hunk 4 above)
Comment (sindhura-kaparthi) [reply .github/workflows/bahmni_metabase_build_publish.yaml]: Yes Done , Thank you

File: package/helm/templates/configMap.yaml (hunk 5 above)
Comment (sindhura-kaparthi) [reply package/helm/templates/configMap.yaml]: Yes Done , Thank you

Comment (mradul1807) [inline package/helm/templates/configMap.yaml]: MB_HOST & MB_PORT are not coming from values.yaml. Review it again.

File: .github/workflows/build_publish.yaml [hunk 6]
```diff
@@ -0,0 +1,77 @@
+name: Build and Publish bahmni-metabase Image
+on:
+  push:
+    branches: 
+      - main
+      - BAH-2665
```
Comment (mohan-13) [inline .github/workflows/build_publish.yaml]: PR looks good. Please remove branch before merge.

**Review by mohan-13 — APPROVED** [review]

File: .github/workflows/build_publish.yaml (hunk 6 above)
Comment (sindhura-kaparthi) [reply .github/workflows/build_publish.yaml]: Sure , We will remove it. Thank you .

**Review by pajmera4 — APPROVED** [review]

**Review by mradul1807 — APPROVED** [review]

File: package/helm/templates/configMap.yaml (hunk 5 above)
Comment (sindhura-kaparthi) [reply package/helm/templates/configMap.yaml]: Yes , We removed it. Thank you

**Pull request merged**